Source/WebKit/UIProcess/API/Cocoa/WKHTTPCookieStore.mm

```diff
@@ -125,6 +125,23 @@ - (void)removeObserver:(id<WKHTTPCookieStoreObserver>)observer
     _cookieStore->unregisterObserver(*result);
 }
 
+- (void)setCookiePolicy:(WKCookiePolicy)policy completionHandler:(void (^)(void))completionHandler
+{
+    auto corePolicy = policy == WKCookiePolicyAllow ? WebCore::HTTPCookieAcceptPolicy::AlwaysAccept : WebCore::HTTPCookieAcceptPolicy::Never;
```

Comment:
This AlwaysAccept needs more investigation.  I'm not sure that's the correct default in all cases.